projects/projen.io/js/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://github.com/projen/projen/archive/refs/tags/v{{version.raw}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: projen/projen
+  strip: /^v/
+
+dependencies:
+  nodejs.org: ^22
```

Comment:
does it strictly require node^22 to run? or can this be relaxed?

Reply:
I can relax it I think.

Reply:
Encountered a warning when relaxing the constraint.

```
This software is currently running on node v23.6.1.
As of the current release of this software, supported node releases are:
  - ^22.0.0 (Planned end-of-life: 2027-04-30)
  - ^20.0.0 (Planned end-of-life: 2026-04-30)
  - ^18.0.0 (Planned end-of-life: 2025-04-30)
```

Reply:
that should be `nodejs.org: ^22 | ^20 | ^18`